frame/merkle-mountain-range/rpc/Cargo.toml

```diff
@@ -17,6 +17,7 @@ codec = { package = "parity-scale-codec", version = "2.0.0" }
 jsonrpsee = { git = "https://github.com/paritytech/jsonrpsee", branch = "master", features = ["server"] }
 serde_json = "1"
 serde = { version = "1.0.126", features = ["derive"] }
+log = "0.4"
```

Comment:
ah so this is what you meant by the issue on `log`? Yeah this is a bit awkward. :/

Reply:
yeah :)